### PR TITLE
ru: Remove WebRTCSidebar macro

### PR DESCRIPTION
**Pull request by nschonni**
### Description

Macro has been removed from the en-US content

### CODE REVIEW

File: files/ru/web/api/media_capture_and_streams_api/taking_still_photos/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ tags:
 translation_of: Web/API/WebRTC_API/Taking_still_photos
 original_slug: Web/API/Media_Streams_API/Taking_still_photos
 ---
-{{WebRTCSidebar}}
+{{DefaultAPISidebar("Media Capture and Streams")}}
```
Comment (nschonni) [inline files/ru/web/api/media_capture_and_streams_api/taking_still_photos/index.md:9]: This one had a different scope in the English that WRTC

File: files/ru/web/api/webrtc_api/adapter.js/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ tags:
 translation_of: Web/API/WebRTC_API/adapter.js
 ---
 
-{{WebRTCSidebar}}
+{{DefaultAPISidebar("WebRTC")}}
```
Comment (nschonni) [inline files/ru/web/api/webrtc_api/adapter.js/index.md:9]: This file is removed in the English, but this might be pending a Sync PR landing here

**Comment by nschonni** [comment]
Sorry to tag the other locales, but was trying to resolve the redirect job failure. Will try and pull it out sepately if I get it working

**Comment by yin1999** [comment]
> Sorry to tag the other locales, but was trying to resolve the redirect job failure. Will try and pull it out sepately if I get it working

No worries. And I'd like to have a review on the redirect fixes for `l10n-zh` :)

**Comment by nschonni** [comment]
@yin1999 I threw those changes in this branch for now https://github.com/nschonni/mdn-translated-content/tree/redirects but I haven't gotten back to it. I think there was a discussion on another issue around some of the issues with the redirects right now

**Review by sashasushko — APPROVED** [review]
Thanks 🙏

**Pull request merged**